Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN set -eux; \
 	chmod +x /usr/local/bin/skaffold; \
 	skaffold version
 
-ARG K8S_VERSION=v1.28.3
+ARG K8S_VERSION=v1.28.4
```

Comment:
While the image update is probably not breaking anything, it would still need to be tested by making the existing `install-test` job use the Action with the new image, e.g. by changing the image reference [here](https://github.com/cloudbees-io/helm-install/blob/v1.1.7/action.yml#L55) at least temporarily.

Reply:
the image is now being tested by the `uses: ./.cloudbees/testing`